scheduler/scheduler.go

```diff
@@ -486,7 +488,9 @@ func (s *scheduler) startTask(id, source string) []serror.SnapError {
 	}
 	// Group dependencies by the node they live on
 	// and subscribe to them.
-	depGroupMap := s.gatherMetricsAndPlugins(t.workflow)
+	depGroupMap := make(map[string]depGroup)
+	depGroupMap[""] = depGroup{Metrics: []core.Metric{}}
```

Comment:
Metrics are no longer needed. I changed it, so no there's new alias type `type depGroupMap map[string][]core.SubscribedPlugin`